TelosLabs/giving-connection
# Feature/embedding youtoube videos

### How to test it:

- Go to an organization that has one location, and edit it. (Admin Panel and User form work)
- Spot the Youtube Video Link Field
- Copy paste any video link from youtoube into that field and Save the organization.
- Go to the organization show page and you should see the video at the end

The video will appear in every location, you can also test that the Youtube Field appears on each location of the form, therefore, if the organization has many locations, there will be a field for each one of them. 

For now, there's a bug, and we can't save organizations with one or more locations
